src/components/Bundlr.tsx

```diff
@@ -80,6 +81,15 @@ export function Bundlr({ children }: { children?: React.ReactNode }) {
                   <NumberIncrementStepper />
                   <NumberDecrementStepper />
                 </NumberInputStepper>
+                <Text
```

Comment:
I'm not a big fan of the absolute positioning here. As the input text can overrun the ETH (I realize in practice i've typed a really long number, but it's still possible)

![image](https://user-images.githubusercontent.com/99622829/199110451-b20bd522-d0b0-4524-b631-63c2376bef11.png)

I would probably not use a `<NumberInput>` as I'm not sure the value in clicking up and down on a value with potentially 18 decimals of accuracy. Change it to and `<Input>` and use `<InputRightElement >`  from chakra to get the ETH overlay.

Reply:
It was my suggestion to put "ETH" over the input, so the user is aware the currency being paid as they type in the input.

I think we should restrict the decimals to 18, and test if it overlaps at that point. User's probably aren't going to be putting in > 1 ETH, so I don't think we need to worry about the left side of the decimal. 

We can handle the overlap issue for now by forcing the input to be a specific min-width, and when we support mobile can figure out a different solution at that point.

Reply:
thanks @mobiusTripper-crypto @sethhrbek  I am now using <Input> and use <InputRightElement> and created a new function to handle change (handleInputChange). Stepper seems available only on <NumberInput> so had to removed it.

I included some checks in the handleInputChange() to verify the number is a valid Wei, and have set the max characters to 40, which about spans the entire box length.
![Screenshot 2022-11-02 at 18 48 54](https://user-images.githubusercontent.com/105349716/199563951-80b3c88c-10ce-472f-a1a9-8f4d133e8778.png)